packages/tools/devtools/devtools-browser-extension/src/popup/PopupView.tsx

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import {
+	DevtoolsFeatures,
+	GetDevtoolsFeatures,
+	type ISourcedDevtoolsMessage,
+	type InboundHandlers,
+	handleIncomingMessage,
+} from "@fluidframework/devtools-core";
+import { type BackgroundConnection } from "../BackgroundConnection";
+
+const queryTimeoutInMilliseconds = 2000; // 2 seconds
```

Comment:
It might be worth adding a quick note to this that the recipient of the message we send is running on the same machine, just in a different process. We aren't waiting on any network requests or anything, so a long wait time shouldn't be necessary. 2 seconds should be more than sufficient for the vast majority of cases.